### ISSUE TITLE
Add onlyImported flag for jsx-no-undef

### ISSUE THREAD

**Issue by nkt** [yannickcr/eslint-plugin-react#922]
There is component named `Text` in my project. ESLint doesn't tell me that `Text` is `undefined` because it's existing class in the [DOM API](https://developer.mozilla.org/en-US/docs/Web/API/Text).
I suggest add rule which deny use components defined globally:

``` json
{
  "rules": {
    "react/jsx-no-undef": ["error", { "onlyImported": true }]
  }
}
```

``` js
import Text from './Text';

<Text /> // ok
```

``` js
<Text /> // react/jsx-no-undef: 'Text' component not found in current scope.
```

**Comment by ljharb** [comment]
This is a great solution to this problem for React.

**Comment by alexzherdev** [comment]
Fixed in #1013 ?
